requirements-test.txt

```diff
@@ -9,4 +9,5 @@ pytest-mock~=3.6
 responses
 testslide~=2.6
 moto~=2.2
-MarkupSafe==2.0.1
\ No newline at end of file
+MarkupSafe==2.0.1
+black==22.1.0
```

Comment:
Why an `==` dependency? Wouldn't we like to at least get minor upgrades?

Reply:
Absolutely! :+1: 
Fixed

Reply:
Realized that I had to fall back to the previous version, as the new stable releases of black require click>8.0 ...
I adjusted the PR description to highlight that issue.